yarn.lock

```diff
@@ -23872,7 +23872,7 @@ reflect.ownkeys@^0.2.0:
   resolved "https://registry.yarnpkg.com/reflect.ownkeys/-/reflect.ownkeys-0.2.0.tgz#749aceec7f3fdf8b63f927a04809e90c5c0b3460"
   integrity sha1-dJrO7H8/34tj+SegSAnpDFwLNGA=
 
-refractor@^3.2.0, refractor@^3.3.1:
+refractor@^3.2.0, refractor@^3.3.1, refractor@^3.4.0:
   version "3.3.1"
```

Comment:
If you bump this to `3.4.0` you might be able to remove `refractor` and `prismjs` from package.json `resolutions`

Reply:
Cleaned up nicely, thanks!